Update README.md (#618)
DNNCommunity/DNNDocs: README.md

Unless you are part of the DNNDocs core team, you will only have read access (you can only **pull**) to this remote.

## Running the DNN Docs Project Locally

> ### .NET Frameworks

Reviewer: ```suggestion
### .NET Framework Prerequisites
```

> ### .NET Frameworks?

Reviewer 2: ```suggestion
### .NET Frameworks
```

I would just remove the question mark...

> You should insure that you already have [.NET Framework v5.0](https://dotnet.microsoft.com/download/dotnet) and the "Developer Pack" for [.NET Framework v4.6.2](https://dotnet.microsoft.com/download/dotnet-framework/net462) install on your machine before trying the build process.

Reviewer: ```suggestion
You should ensure that you already have [.NET 5.0](https://dotnet.microsoft.com/download/dotnet) (used by the `build` project) and the "Developer Pack" for [.NET Framework v4.6.2](https://dotnet.microsoft.com/download/dotnet-framework/net462) (used by the custom `DocFx` plugins) installed on your machine before trying the build process.
```
